tiup/tiup-cluster-topology-reference.md

```diff
@@ -0,0 +1,769 @@
+
+For the above fields, you cannot modify these configured fields after the deployment:
+
+- `host`
+- `port`
+- `deploy_dir`
+- `data_dir`
+- `log_dir`
+- `arch`
+- `os`
+
+A `monitoring_servers` configuration example:
+
+```yaml
+monitoring_servers:
+  - host: 10.0.1.11
+    rule_dir: /local/rule/dir
+```
+
+### `grafana_servers`
+
+`grafana_servers` specifies the machines on which the Grafana services are deployed. It also specifies the service configuration on each machine. `grafana_servers` is an array. Each array element contains the following fields:
+
+- `host`: Specifies the machine on which the Grafana services are deployed. The field value is the IP address and cannot be omitted.
+
+- `ssh_port`: The SSH port used when connecting to the target machine for operations. If it is not specified, the `ssh_port` of the `global` section is used.
+
+- `port`: The listening port of the Grafana services. The default value is `3000`.
+
+- `deploy_dir`: Specifies the deployment directory. If it is not specified or specified as a relative directory, the directory is generated according to the `deploy_dir` directory configured in `global`.
+
+- `os`: The operating system of the machine specified in `host`. If this field is not specified, the default value is the `os` value in `global`.
+
+- `arch`: The architecture of the machine specified in `host`. If this field is not specified, the default value is the `arch` value in `global`.
+
+- `username`: The user name on the Grafana login interface.
+
+- `password`: The password corresponding to Grafana.
+
+- `dashboard_dir`: This field specifies a local directory that should contain complete `dashboard(*.json)` files. These files are transferred to the target machine during the initialization phase of the cluster configuration as the dashboards for Grafana.
+
+- `resource_control`: Resource control for the service. If this field is configured, the field content is merged with the `resource_control` content in `global` (if the two fields overlap, the content of this field takes effect). Then, a systemd configuration file is generated and sent to the machine specified in `host`. The configuration rules of `resource_control` are the same as the `resource_control` content in `global`.
+
+> **Note:**
+>
+> If the `dashboard_dir` field of `grafana_servers` is configured, after executing the `tiup cluster rename` command to rename the cluster, you need to perform the following operations:
+>
+> + For the `*.json` files in the local dashboards directory, update the value of the `datasource` field to the new cluster name (because `datasource` is named after the cluster name).
+> + Execute the `tiup cluster reload -R grafana` command.
```

Comment:
> 
```suggestion
> 1. For the `*.json` files in the local dashboards directory, update the value of the `datasource` field to the new cluster name (because `datasource` is named after the cluster name).
> 2. Execute the `tiup cluster reload -R grafana` command.
```